### DISCUSSION TITLE
Parte 2, códigos de ejemplo

### DISCUSSION THREAD

**Discussion by MaxAl100** [IIC2133-PUC/2024-1#177] [Tarea 3] [answered]
Tengo una duda respecto a los códigos de ejemplo dados en el pdf para la parte 2. Específicamente para el segundo y tercer ejemplo.
En el segundo está el basural de tamaño 2 repetido, por lo que este no se considera dentro de los 5 basurales distintos (hay más de 5 en la línea)
![image](https://github.com/IIC2133-PUC/2024-1/assets/43099597/71af64fe-028b-42fd-9ec0-a3bc629d7317)

Por otro caso, en el tercer ejemplo se debiesen recibir 6 basurales distintos, pero uno está repetido, por lo que hay solo 5. 
![image](https://github.com/IIC2133-PUC/2024-1/assets/43099597/c7c209a7-b6d4-415a-8d4a-8e616f493a24)

Entonces mi duda es si hay que considerar los repetidos o si hay que eliminarlos de la lista de información, ya que en un caso se ignoran y en el otro no se pueden ignorar debido a que causan un problema de falta de bodegas...

**Comment by benjavicente** [comment, marked as answer]
El primero es un error, debería ser 7, gracias por notarlo rápido. Subí el arreglo del enunciado. El segundo está bien. Decimos que son basurales distintos (piénsalo como lugares distintos), pero  algunos de esos basurales aportan la misma cantidad (mismo tipo de aporte).

> Entonces mi duda es si hay que considerar los repetidos o si hay que eliminarlos de la lista de información, ya que en un caso se ignoran y en el otro no se pueden ignorar debido a que causan un problema de falta de bodegas...

No me quedó muy claro a lo que te refieres. Lo importante es que cualquiera se puede usar varias veces, y debes ver la cantidad mínima de usos para llegar al objetivo. Probablemente, ese tipo de consideraciones sea parte la solución 🌚

**Reply by MaxAl100** [reply]
La duda era porque en el ejemplo 2 al "eliminar" (ignorar) los repetidos se llegaba a 5 basurales, lo que me generaba la duda. Pero al tener que ser 7 ahí me queda claro

**Reply by MaxAl100** [reply]
Hola, me parece que no realizaron el cambio en el pdf 😅

**Reply by benjavicente** [reply]
Subido.

**Reply by MaxAl100** [reply]
Ahora sí, confirmo! 👍
